torch/quantization/fx/convert.py

```diff
@@ -188,6 +185,10 @@ def convert(model: GraphModule, is_reference: bool = False,
         weight_eq_obs_dict = update_obs_for_equalization(model, modules)
         convert_eq_obs(model, modules, weight_eq_obs_dict)
 
+    # always run weight observers in the top level forward method
+    # for dynamic quant ops or weight only quant ops
+    run_weight_observers(model)
```

Comment:
Why did we move this here?

Reply:
In response to this chain: https://github.com/pytorch/pytorch/pull/59963#discussion_r653953913

tdlr: We would need to run the weight observers again after the equalization because of the newly scaled weights to update the qparams, so I moved the function call to down here.